code-security/admin_guide/scan-monitor/iac-tag-and-trace.adoc

```diff
@@ -0,0 +1,64 @@
+
+== IaC Tag and Trace
+
+Prisma Cloud Code Security supports infrastructure-as-code (IaC) tags that help you trace the link for resources that are deployed from code to your cloud infrastructure. 
+Using Yor, an open-source auto-tagging tool that supports Terraform, CloudFormation, and Serverless, you can adds tags to all resource blocks in your repository. 
+Yor uses YAML configuration to tag and trace resources, and these automated tags are unique to a repository. 
+Yor automatically creates additional tags that contain details such as:
+
+* Git organization, repository, and the exact file that contains the infrastructure-as-code (IaC) template used to create the cloud resource.
+* Timestamp of Git commits for supported cloud resources that includes the list of all editors and contributors who modified the file.
+* Custom tags that you create for your resource. 
++
+These tags can be as simple as key and value pairs or a complex combination, yet Yor can auto-detect them. You can edit custom tags by detailing a tag rule on the Prisma Cloud console. Tag rule is helpful to define resources that impact the cost and ownership of any third-party resources within your environment.
+
+If you do have an existing tag management strategy you can choose to replicate it using tag rules on the Prisma Cloud console.
+
+In addition to tagging, you can use yor_trace to detect drift in code and identify teams and resource owners to help triage a fix in the most time- and cost-effective way. Yor trace creates a unique tag for every runtime resource, which helps you locate the specific resource within a commit.  
```

Comment:
Should this be, "In addition to the Git tags, each tagged resource is also assigned a yor_trace tag that is unique to every runtime resource. You can use yor_trace to ....